data/mt/retired/Barry-Usher-69cfa3aa-7091-4752-a99c-b6a42c70b17b.yml

```diff
@@ -7,7 +7,8 @@ email: barry.usher@mtleg.gov
 party:
```

Comment:
@bfossen-ce this person was elected to the state senate and begins on 01/01/2023 and had an entry earlier in this pull. Want to make sure that this isn't duplicative.